modules/CONT-02_Using-branches.yml

```diff
@@ -33,7 +33,7 @@ screens:
       presenter-script:
         - Let's practice creating a branch.
       steps:
-        - description: "Create a branch in the repo named firstname-lastname (using your first and last names)."
+        - description: "Create a branch in your repo and name it firstname-lastname using your first and last name."
```

Comment:
> Create a branch in **your** repo

My only issue with this change is that it is solely focused on self-serve. In the instructor led classes they will not have individual repos and they should create their branch in the class repo. This was the point of adding the class repo link to help them get to the right place. Perhaps we could say something like: "Create a branch in the class repo (linked above)"